lib/rspec/matchers/built_in/raise_error.rb

```diff
@@ -160,6 +160,7 @@ def warn_about_bare_error
                         "will match when Ruby raises a `NoMethodError`, `NameError` or " \
                         "`ArgumentError`, potentially allowing the expectation to pass " \
                         "without even executing the method you are intending to call. " \
+                        "Actual error raised was #{description_of(@actual_error)}. "\
```

Comment:
@xaviershay and @myronmarston I tried to write a test for when `@actual_error` is `nil` and I may be doing something wrong, can I please get some pointers on how to rewrite this test?

```
it 'issues a warning that does not include current error when its not present' do
  expect(::Kernel).to receive(:warn) do |message|
    ex = /Actual error raised was/
    expect(message).not_to match ex
  end
  expect{ '' }.to(raise_error())
 end
```